lib/runtime-core/examples/simple/main.rs

```diff
@@ -1,16 +1,46 @@
 use wabt::wat2wasm;
 use wasmer_clif_backend::CraneliftCompiler;
-use wasmer_runtime_core::{error::Result, prelude::*};
+use wasmer_runtime_core::{
+    error::Result,
+    global::Global,
+    memory::Memory,
+    prelude::*,
+    table::Table,
+    types::{ElementType, MemoryDesc, TableDesc, Value},
+};
 
 static EXAMPLE_WASM: &'static [u8] = include_bytes!("simple.wasm");
 
 fn main() -> Result<()> {
     let wasm_binary = wat2wasm(IMPORT_MODULE.as_bytes()).expect("WAST not valid or malformed");
     let inner_module = wasmer_runtime_core::compile_with(&wasm_binary, &CraneliftCompiler::new())?;
 
+    let memory = Memory::new(MemoryDesc {
+        min: 1,
+        max: Some(1),
+        shared: false,
+    })
+    .unwrap();
+
+    let global = Global::new(Value::I32(42));
+
+    let table = Table::new(TableDesc {
+        ty: ElementType::Anyfunc,
```

Comment:
Could `ty` be `element`? ref: https://github.com/WebAssembly/spec/blob/cf67a477d2d802329640ef4e6461e4c180d8f910/test/js-api/table/constructor.any.js#L57

Reply:
Okay, I agree with this.